frontend/src/components/admin/generalprogramme/GeneralProgrammeDashboard.js

```diff
@@ -0,0 +1,489 @@
+    if (selectedProgramme) {
+      fetchCapturedData(
+        selectedProgramme.id,
+        searchCriteria.startDate,
+        searchCriteria.endDate,
+      );
+    }
+  };
+
+  const handlePageChange = (pageInfo) => {
+    setPage(pageInfo.page);
+    setPageSize(pageInfo.pageSize);
+  };
+
+  const programmeHeaders = [
+    {
+      key: "value",
+      header: intl.formatMessage({ id: "program.name.label" }),
+    },
+    {
+      key: "actions",
+      header: intl.formatMessage({ id: "label.button.action" }),
+    },
+  ];
+
+  const capturedDataHeaders = [
+    {
+      key: "labNumber",
+      header: intl.formatMessage({ id: "sample.label.labnumber" }),
+    },
+    {
+      key: "patientName",
+      header: intl.formatMessage({ id: "patient.label.name" }),
+    },
+    {
+      key: "collectionDate",
+      header: intl.formatMessage({ id: "sample.label.collectiondate" }),
+    },
+    {
+      key: "status",
+      header: intl.formatMessage({ id: "label.status" }),
+    },
+    {
+      key: "actions",
+      header: intl.formatMessage({ id: "label.button.action" }),
+    },
+  ];
+
+  const programmeRows = programmeData
+    .slice((page - 1) * pageSize, page * pageSize)
+    .map((programme) => ({
+      id: programme.id,
+      value: programme.value,
+      actions: programme.id,
+    }));
+
+  const capturedDataRows = capturedData
+    .slice((page - 1) * pageSize, page * pageSize)
+    .map((item) => ({
+      id: item.id,
+      labNumber: item.labNumber || "",
+      patientName: item.patientName || "",
+      collectionDate: item.collectionDate || "",
+      status: item.status || "",
+      actions: item.id,
+    }));
+
+  if (selectedProgramme) {
+    return (
+      <div className="adminPageContent">
+        <PageBreadCrumb breadcrumbs={breadcrumbs} />
+        <Grid fullWidth={true}>
+          <Column lg={16} md={8} sm={4}>
+            <Section>
+              <div
+                style={{
+                  display: "flex",
+                  alignItems: "center",
+                  gap: "1rem",
+                  marginBottom: "2rem",
+                }}
+              >
+                <Button kind="secondary" onClick={handleBackToList}>
+                  <FormattedMessage id="label.button.back" />
+                </Button>
+                <Heading>
+                  {selectedProgramme.name} -{" "}
+                  <FormattedMessage id="program.captured.data.label" />
+                </Heading>
+              </div>
+
+              {/* === SEARCH FILTERS === */}
+              <div
+                style={{
+                  display: "flex",
+                  gap: "1rem",
+                  alignItems: "end",
+                  marginBottom: "2rem",
+                  padding: "1rem",
+                  backgroundColor: "#f4f4f4",
+                  borderRadius: "4px",
+                }}
+              >
+                <DatePicker dateFormat="d/m/Y" datePickerType="single">
```

Comment:
There's a `CustomDatePicker` element that it seems like you should use as it at least supports two date displays.

Reply:
I think this goes for all the other DatePicker's in this PR.